pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -52,6 +52,11 @@ func init() {
 }
 
 func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
+	hostOnlyCIDR = strings.Trim(hostOnlyCIDR, "'\"")
```

Comment:
where is hostOnlyCIDR defined? 

I feel like this fix should move to `start_flags.go`, where the flag processing happens.

Reply:
https://github.com/kubernetes/minikube/blob/f71e525940fede6ef6ad2c5825dba287d0093c09/cmd/minikube/cmd/start_flags.go#L493
I feel like this is where it should be added, please correct me if I'm wrong